app/views/devise/two_factor_authentication_setup/index.html.slim

```diff
@@ -10,5 +10,6 @@ h1.heading = t('devise.two_factor_authentication.two_factor_setup')
   p#2fa-description = t('devise.two_factor_authentication.otp_setup')
   = f.input :phone, as: :tel, label: 'Phone Number', required: true,
       input_html: { 'aria-describedby' => '2fa-description' }
+  = f.input :phone_sms_enabled, as: :boolean, label: 'SMS Enabled?', input_html: { checked: true }
```

Comment:
Most sites that I know never automatically send the code to you. They make you click a button to confirm where you want the code sent. I will submit a PR to the 2FA gem, which will allow us to remove the `phone_sms_enabled` field whose name is misleading since a phone can be sms enabled but should be able to receive a call.